integration/auth/user_mgt_test.go

```diff
@@ -445,14 +453,25 @@ func TestCreateUserMFA(t *testing.T) {
 		t.Fatalf("CreateUser() = %v; want = nil", err)
 	}
 	defer deleteUser(user.UID)
-	var factor []*auth.MultiFactorInfo = []*auth.MultiFactorInfo{
+	var factors []*auth.MultiFactorInfo = []*auth.MultiFactorInfo{
 		{
-			UID:                 user.MultiFactor.EnrolledFactors[0].UID,
-			DisplayName:         "Spouse's phone number",
-			FactorID:            "phone",
-			PhoneNumber:         "+11234567890",
```

Comment:
I think `PhoneNumber` should be in at least one test case, right?

Reply:
Yes, this is the expected output. The idea is, on parsing the server side output, we will always display `PhoneMultiFactorInfo` and not `PhoneNumber`. `PhoneNumber` is just for input since we want to deprecate it.